docs/sources/source/nlp/bert_pretraining.rst

```diff
@@ -191,7 +192,8 @@ For training from raw text use nemo_nlp.nm.data_layers.BertPretrainingDataLayer,
             # data_layer = nemo_nlp.BertPretrainingPreprocessedDataLayer(
```

Comment:
should it be nemo_nlp.nm.data_layers.BertPretrainingPreprocessedDataLayer ?

Reply:
this is for the on the fly processing case, which is done by BertPretrainingDataLayer.